va/va_enc_hevc.h

```diff
@@ -45,6 +45,200 @@ extern "C" {
  * @{
  */
 
+/** Attribute value for VAConfigAttribEncHEVCFeatures.
+ *
+ * This attribute decribes the supported features of an HEVC/H.265
+ * encoder configuration.
+ *
+ * All of the field values in this attribute are VA_FEATURE_* values,
+ * indicating support for the corresponding feature.
+ */
+typedef union VAConfigAttribValEncHEVCFeatures {
+    struct {
+        /** Separate colour planes.
+         *
+         * Allows setting separate_colour_plane_flag in the SPS.
+         */
+        uint32_t separate_colour_planes     : 2;
```

Comment:
If from driver side, this value should be 0, cannot be set to 1,  how to set this field?  The meaning of 0 and 1 are identical to Recommendation ITU-T H.265

Reply:
Yes.  `VA_FEATURE_NOT_SUPPORTED` from the driver indicates that the separate colour planes are not usable, so `separate_colour_planes` must then always be set to zero by the user.